src/app/sidebar/sidebar.component.html

```diff
@@ -29,7 +29,7 @@ <h5>Master Branch </h5>
       </li>
       <li *ngIf="vendor == 'false'" class="item cursor-pointer rounded-left button text-center mt-5 cicd">
         <a href="https://app.mayaonline.io/mdape2e" target="_blank" class="border-0" placement="bottom"
-        ngbTooltip="Click To View Enterprise Pipelies">
+        ngbTooltip="Click here to view Enterprise Pipelines">
```

Comment:
Can you put this text in language file, So that from next time, no need to change at two places!!